src/main/target/RG_SSD_F3/target.h

```diff
@@ -122,7 +122,7 @@
 #define CURRENT_METER_ADC_PIN    PA5
 #define VBAT_SCALE_DEFAULT       119
 
-#define TRANSPONDER
+#define USE_TRANSPONDER
```

Comment:
Can you rebase this please?

Reply:
I think it's done with dd89ba1.

Reply:
dd89ba1 merges master into your branch, and will show up as 'Merge branch 'master' into bfdev-rg_ssd_f3-remove-stale-defs' in master once the pull request is merged. Instead of mergeing master into your branch, please use `git rebase -i master` to achieve the same effect, but keeping a clean commit log.

Reply:
Da** !!!

I'll try that next time.

Actually, I don't have a good experience with `git rebase -I master`. It bit me several times when I was a beginner...

Reply:
What in particular was it that bit you? In general it's quite docile if you thread carefully and don't go overboard in editing the execution plan.

Reply:
I don't really remember... Just a bad memory 😓

Reply:
Try again. It's not that bad. If there's a lot of commits to be squashed, and they throw conflicts, then sometimes it can help to first squash all of them into one commit by rebasing onto the branching point, and then rebase the resulting commit onto the tip of master in a second rebase.

Reply:
Keep that for next time. Just a simple `git rebase -i master` will do for this one.